test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -0,0 +1,268 @@
+﻿
+# Test Settings:
+# This is the list of PowerShell Core modules for which we test update-help
+$moduleList = @(
+    "Microsoft.PowerShell.Archive"
+    "Microsoft.PowerShell.Host"
+    "Microsoft.PowerShell.Core"
+    "Microsoft.PowerShell.Diagnostics"
+    "Microsoft.PowerShell.Management"
+    "Microsoft.PowerShell.Security"
+    "Microsoft.PowerShell.Utility"
+    "PackageManagement"
+    "PowershellGet"
+    "PSReadline"
+)
+
+# The file extension for the help content on the Download Center.
+# For Linux we use zip, and on Windows we use $extension.
+$extension = ".zip"
+
+if ([System.Management.Automation.Platform]::IsWindows)
+ {
+    $moduleList += @(
+        "CimCmdlets"
+        "Microsoft.WsMan.Management"
+        "Microsoft.PowerShell.LocalAccounts"
+    )
+    $extension = ".cab"
+ }
+
+$expectedHelpFiles = @{}
```

Comment:
A thought about maintainability here:
$expectedHelpFiles, $expectedHelpInfoFiles, and $expectedCompressedFiles form a triplet where each module must have an entry in each of the three hashes? If someone adds a new entry, they must do it in all three structures.
While reading through the test, I started wondering if it made sense to organize the info by triplet:
$expectedHelpFiles{"CimCmdlets"] = @("Microsoft.Management.Infrastructure.CimCmdlets.dll-help.xml", "CimCmdlets_fb6cc51d-c096-4b38-b78d-0fed6277096a_HelpInfo.xml", "CimCmdlets_fb6cc51d-c096-4b38-b78d-0fed6277096a_en-US_HelpContent$extension")
$expectedHelpFiles["Microsoft.PowerShell.Archive"]= @( . . .
Alternative: Each could point to a hash instead of an array to provide named access.
Thoughts? Would it make the code easier to read and maintain or more difficult?